src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
+  single menu action. It is already possible, but the syntax not easy for
+  begginers (and non-programmers in general).
+
+  <hrule>
+
+  Being able to adjust the math typesetting parameters. For example, by
+  default, one may find formulas too compact, and would like to have more
+  space around (in)equality signs.
+
+  <hrule>
+
+  A variant for underbraces and overbraces with <verbatim|\\smash> ;
+  currently <verbatim|\<less\>resize\<gtr\>> can be used but it is slow to
+  use and content can't be really modified afterwards.
+
+  <hrule>
+
+  A set of shortcuts for common colors (red, dark green, blue, grey).
+
+  <hrule>
+
+  A set of shotcuts for commonly used unicode symbols in text mode, such as
+  \<Rightarrow\>, \<rightarrow\>, <text-dots> similar to shortcuts that exist
+  in math mode.
+
+  <hrule>
+
+  The "draw image" feature is heavily bugged. It may be wiser to not loose
+  developement time on it. This may be the job of external programs such as
+  Inkscape. A useful feature would be to place inline formulas and text
+  floating (with relative positions/sizes) on top of figures, movable by
+  mouse. This would eliminiate the use of, for example, latexit in Inkscape.
```

Comment:
This is already possible. See the feature "Ink here" under the "Draw" menu. You can insert graphs on other content, in particular you can insert text and formulas at arbitrary positions.